src/jobflow_remote/cli/types.py

```diff
@@ -266,6 +266,15 @@
     ),
 ]
 
+yes_opt = Annotated[
+    bool,
+    typer.Option(
+        "--yes",
+        "-y",
+        help="Sets any confirmation values to 'yes' automatically",
```

Comment:
Maybe this ?
```suggestion
        help="Automatic yes to prompts; assume 'yes' as answer to all prompts and run non-interactively.",
```
Just a very tiny detail. (this is exactly what is in the doc of apt-get)

Reply:
I would keep as it is for the moment (for the record, copied by the help of the conda command...), as it is not entirely accurate. In case the worker has a 2FA the password will be asked, so it will not be "non-interactive". I propose to revise this description in the PR where we switch to the `--yes` option for all the commands and we have a better idea of when it gets applied.

Reply:
sure ok for me!